src/graphql/resolvers/mutations/session.js

```diff
@@ -23,24 +23,43 @@ export const fieldResolvers = {
       dlog('TCL: events %o', userEvents);
       dlog('SessionMutation:update called');
 
-      const [sessionResults, userResults] = await Promise.all([
+      // we need the original before we update it.
+      const orginalSession = await sessionStore(
+        firestore,
+        logger,
+      ).findMySession({
+        user,
+        sessionId,
+      });
+
+      const [updatedSession, userResults] = await Promise.all([
         sessionStore(firestore, logger).update({
           user,
           sessionId,
           session,
         }),
 
-        memberStore(firestore, logger).find(user.sub),
+        memberStore(firestore).find(user.sub),
       ]);
 
-      if (sessionResults.status === 'ACCEPTED') {
+      if (
+        orginalSession.status === 'DRAFT' &&
+        updatedSession.status === 'SUBMITTED'
+      ) {
+        userEvents.emit('newSessionCreated', {
+          user: userResults,
+          session: updatedSession,
+        });
+      }
+
+      if (updatedSession.status === 'ACCEPTED') {
```

Comment:
If a session is updated after it is ACCEPTED, will another accepted email go out?

Reply:
right now yes. but we don't say what was changed, etc.. which got me thinking about scheduling. maybe we pull it for now until we have something better. it's kinda like telling the user, yea our db save worked.. meh..